public/app/features/canvas/elements/icon.tsx

```diff
@@ -60,6 +60,7 @@ export function IconDisplay(props: CanvasElementProps) {
       src={data.path}
       style={svgStyle}
       className={svgStyle.strokeWidth ? svgStrokePathClass : undefined}
+      cleanStyle={true}
```

Comment:
Hi just checking in from the alps 😬 we might want to also test other places we use custom svgs (like geomap) great job finding a viable fix!